mediacenter/templates/mediacenter/document_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'one-row.html' %}
+{% load i18n %}
+
+{% block row %}
+    <h2>{% blocktrans %}Are you sure you want to delete document "{{ document }}"?{% endblocktrans %}</h2>
```

Comment:
I always prefer to use `document.title` than to refer to the `__unicode__` method, but it's my point of view. Explicit is better than implcit. By using the field, I don't need to go to the model to see what will be displayed